Legge/wavreader module & role (#966)
NUbots/NUbots: module/sound/SpeechRecognition/README.md

> SpeechRecognition
> =================

Reviewer: ```suggestion
# SpeechRecognition
```

## Description

> This module takes in English speech as a wav file and converts it into text. It uses vosk-api to decipher the audio into text.

Reviewer: ```suggestion

This module takes in English speech as binary data and converts it into text. It uses vosk-api to decipher the audio into text.
```
Newline for consistency with other readmes. This module takes binary audio data, it isn't specific to the WavReader module.

## Usage

## Consumes

> Uses an Audio Data message.

Reviewer: ```suggestion

- message::input::AudioData with the audio data to convert to text
```
